processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -244,22 +243,97 @@ public int lookupId(String name)
     };
   }
 
+  public static class DimensionDesc
```

Comment:
The unofficial convention is to put static classes at the end of class definitions, if I recall from @himanshug 's formatting requests on other PRs